src/index.ls

```diff
@@ -44,7 +44,7 @@ exports <<<
                   output = new SourceNode null, null, null, [
                       "// Generated by LiveScript #{exports.VERSION}\n", output
                   ]
-              if options.map and options.map isnt 'none'
+              if options.map? and options.map isnt 'none'
```

Comment:
Yes, that is right, `options.map` should be a string, 'none', 'embedded', etc. If you now passed `options.map = 5`, it would be as if it were `embedded`, my change did not effect this behavior. I only changed it because I thought it to be better LiveScript code to use `value?` for checking the existence of some value, because it makes the intent clear.

If the code is only for control flow, than how it currently is should suffice. Then you assume the value to be valid. If you also want to do validation, the check would become:`if options.map in <[ linked linked-src embedded debug ]>`. But than you will have to change a lot more code, because validation of options is currently not being done except for the CLI.